website/content/en/v0.32/concepts/nodepools.md

```diff
@@ -283,8 +281,6 @@ kubelet:
   maxPods: 20
 ```
 
-You can specify the container runtime to be either `dockerd` or `containerd`. By default, `containerd` is used.
-
 * `containerd` is the only valid container runtime when using the `Bottlerocket` AMIFamily or when using Kubernetes version 1.24+ and the `AL2`, `Windows2019`, or `Windows2022` AMIFamilies.
```

Comment:
Can we drop this bullet here too?

Reply:
@jonathan-innis not sure I get it, you mean removing the whole
`* `containerd` is the only valid container runtime when using the `Bottlerocket` AMIFamily or when using Kubernetes version 1.24+ and the `AL2`, `Windows2019`, or `Windows2022` AMIFamilies.` block?

Reply:
Yep, since you can't specify the field at all now, I don't think that it makes sense to have mention of it.

Reply:
> Yep, since you can't specify the field at all now, I don't think that it makes sense to have mention of it.

LOL, I initially did that and then reverted back to. Fixed